### ISSUE TITLE
Only English speech recognition

### ISSUE THREAD

**Issue by antoniusostermann** [api-ai/api-ai-android-sdk#23]
I tried to follow the presented tutorial, so I'm currently having a very simple AI-application. Now I changed the supported language to German, but still the app only recognizes english words. This is my configuration:

```java
final AIConfiguration config = new AIConfiguration("---", AIConfiguration.SupportedLanguages.German, AIConfiguration.RecognitionEngine.System);
```

My dependencies:
```gradle
compile 'com.android.support:appcompat-v7:25.0.1'
testCompile 'junit:junit:4.12'
compile 'ai.api:sdk:2.0.0@aar'
compile 'ai.api:libai:1.2.1'
compile 'com.android.support:appcompat-v7:25.0.1'
compile 'com.google.code.gson:gson:2.8.0'
compile 'commons-io:commons-io:20030203.000550'
```

**Comment by a23sokolov** [comment]
Thank you for information, the issue is confirmed. It will be fixed in a closest future. I'll notify you after.

**Comment by a23sokolov** [comment]
Source updated. You can use it or wait today lib update.

**Comment by xVir** [comment]
Hi, @antoniusostermann 
Library with fix of the issue published, version 2.0.1
Thanks!
